cmd/kubeadm/app/master/apiclient.go

```diff
@@ -75,7 +76,12 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 func WaitForAPI(client *clientset.Clientset) {
 	start := time.Now()
 	wait.PollInfinite(kubeadmconstants.APICallRetryInterval, func() (bool, error) {
-		// TODO: use /healthz API instead of this
+		healthStatus := 0
```

Comment:
It seems we want to do something like this instead of what using `ComponentStatuses`, not both of them.